protobuf/ha_cluster_agent.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
```

Comment:
Shall we consider this API also versioned and put it inside v1 scope ?

Reply:
I think so but will this api be used by the data plane? If this is for the control-plane only it should go here: https://github.com/openebs/mayastor-control-plane/tree/develop/control-plane/grpc/proto/v1

Reply:
I have added this changes to control-plane PR.